packages/components/src/InputGroup/InputGroup.tsx

```diff
@@ -0,0 +1,35 @@
+import React, { ReactElement } from "react";
+import classnames from "classnames";
+import styles from "./InputGroup.css";
+
+interface InputGroupProps {
+  readonly children: ReactElement | ReactElement[];
+  readonly flowDirection?: "horizontal" | "vertical";
+}
+
+export function InputGroup({
+  children,
+  flowDirection = "vertical",
+}: InputGroupProps) {
+  if (checkChildren(children)) return <></>;
+
+  const className = classnames(styles.inputGroup, styles[flowDirection]);
+  return <div className={className}>{children}</div>;
+}
+
+function checkChildren(childs: ReactElement | ReactElement[]): boolean {
+  return React.Children.toArray(childs).some(child => {
+    if (
+      child.type === InputGroup &&
+      child.props.flowDirection != "horizontal"
+    ) {
+      console.error(
```

Comment:
@seanhealy By making this a console *error* won't we trigger JS error reporting (like rollbar)? That could be noisy.

Reply:
Only on production. We probably also want to know about it if it's happening there.